drivers/at/at.c

```diff
@@ -86,6 +133,11 @@ int at_recv_bytes_until_string(at_dev_t *dev, const char *string,
 {
     size_t len = 0;
     char *_string = (char *)string;
+    int res = 0;
+
+#if IS_USED(MODULE_AT_URC_ISR)
+    dev->awaiting_response = true;
+#endif
 
     while (*_string && len < *bytes_len) {
         int res;
```

Comment:
res is redeclared here but is not used afterwards in the while loop.

Reply:
Good catch. Fixed